test/unit/process/test_arrhenius_rate_constant.cpp

```diff
@@ -7,40 +7,43 @@ TEST(ArrheniusRateConstant, DefaultConstructor){
   micm::ArrheniusRateConstant arrhenius{};
 }
 
-TEST(ArrheniusRateConstant, CalculateWithSystem){
-  micm::ArrheniusRateConstant zero{};
-  auto k = zero.calculate(micm::System());
-  EXPECT_NEAR(k, 1, 0.01);
+//
+// TODO: jiwon 5/18 - commented out because System doesn't have default constructor anymore
```

Comment:
@mattldawson initially we had said rate constants would calculate from a system. Should this now really be a state?

Reply:
yes, I think so - I have been making this change for my issue also

Reply:
@K20shores Could you elaborate what changes need to be made?

Reply:
Actually, this can be done in a future task. I'll make a ticket for it.